velox/core/tests/TestPlanFragment.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gtest/gtest.h>
+
+#include "velox/core/Context.h"
+#include "velox/core/PlanFragment.h"
+#include "velox/core/QueryConfig.h"
+#include "velox/core/QueryCtx.h"
+
+using namespace ::facebook::velox;
+using namespace ::facebook::velox::core;
+
+namespace {
+std::shared_ptr<QueryCtx> getSpillQueryCtx(
+    bool spillEnabled,
+    bool aggrSpillEnabled,
+    bool joinSpillEnabled,
+    bool orderBySpillEnabled) {
+  std::unordered_map<std::string, std::string> configData({
+      {QueryConfig::kSpillEnabled, spillEnabled ? "true" : "false"},
+      {QueryConfig::kAggregationSpillEnabled,
+       aggrSpillEnabled ? "true" : "false"},
+      {QueryConfig::kJoinSpillEnabled, joinSpillEnabled ? "true" : "false"},
+      {QueryConfig::kOrderBySpillEnabled,
+       orderBySpillEnabled ? "true" : "false"},
+  });
+  return std::make_shared<QueryCtx>(
+      nullptr, std::make_shared<MemConfig>(configData));
+}
+}; // namespace
+
+TEST(TestPlanFragment, basic) {
```

Comment:
@spershin @xiaoxmeng TestPlanFragment -> PlanFragmentTest per naming convention. Same for the file name. Some old files do not follow this convention, but new code should. Please, update.